src/auto-import.js

```diff
@@ -47,7 +47,7 @@ const existingImport = (importNode, identifier, { style }) => {
       }
 
       if (style === 'default') {
-        const range = [importNode.start + 6, importNode.start + 7];
+        const range = [importNode.range[0] + 6, importNode.range[0] + 7];
```

Comment:
'Use node.range[0] instead of node.start'

'Use node.range[1] instead of node.end'

https://github.com/eslint/eslint/blob/main/lib/rule-tester/rule-tester.js#L237